atd-etl/app/process_hasura_import.py

```diff
@@ -84,21 +85,70 @@ def process_line(file_type, line, fieldnames, current_line, dryrun=False):
 
         # If not found, then insert
         if record_existing is None:
-            insert_crash = True
+            insert_record = True
         # Else, record exists we need to compare
         else:
             print("[%s] Exists: %s (%s)" % (str(current_line), str(crash_id), file_type))
             existing_records += 1
-
-            insert_crash = record_crash_compare(
+            # Determine if we need to update...
+            insert_record = record_crash_compare(
                 line=line,
                 fieldnames=fieldnames,
                 crash_id=crash_id,
                 record_existing=record_existing
             )
 
-    # The record does not exist, insert.
-    if file_type != "crash" or insert_crash:
+    # Other file types
+    if file_type != "crash":
+        # First, we retrieve the current crash id from the plain-text CSV line
+        crash_id = get_crash_id(line)
+        # Now check if the current file type is one of these tables
+        if file_type in ["unit", "person", "primaryperson"]:
```

Comment:
should we include "charges" here too?

Reply:
Good question, initially it was only those three categories. @JaceDeloney should charges also be part of the bundle or should they be updated automatically?